web/src/components/table/use-cases/sessions.tsx

```diff
@@ -176,8 +176,10 @@ export default function SessionsTable({
     {
       accessorKey: "userIds",
       enableColumnFilter: !omittedFilter.find((f) => f === "userIds"),
+      id: "userIds",
```

Comment:
<img width="1346" alt="image" src="https://github.com/langfuse/langfuse/assets/17686849/5fa1ab9b-2116-4f6f-b6ec-c8f3ed9c5eee">
I just saw that in our seed:load command we end up in a state where we have a sessionid with traces null. Can you check what is going on to me? It should be at least 1 in my understanding. Can be another ticket as this is not really related to this change.

Reply:
After short investigation I believe that this session didn't get created by the seeder as the syntax of the seeder script is: `session-${i % 10}`.